README.mdown

```diff
@@ -39,6 +41,30 @@ curl -X POST -H "Content-Type: application/json" \
      http://127.0.0.1:8080/incoming/your-secret-token
 ```
 
+### UptimeRobot
+
+The URL is constructed as follows: `http(s)://<host>:<port>/incoming/uptimerobot/<your-secret-token>`
+
+#### Usage
+
+Send a GET request to the webhook URL with the follow query params.
```

Comment:
This is a very confusing opener. Using this does not mean having to send a GET request somewhere.